applications/ShapeOptimizationApplication/custom_utilities/mapping/mapper_vertex_morphing.h

```diff
@@ -508,7 +508,7 @@ class MapperVertexMorphing : public Mapper
             double sum_of_weights = 0.0;
 
             if(number_of_neighbors >= max_number_of_neighbors)
-                std::cout << "\n> WARNING!!!!! For node " << node_i.Id() << " and specified filter radius, maximum number of neighbor nodes (=" << max_number_of_neighbors << " nodes) reached!" << std::endl;
+                KRATOS_WARNING("ShapeOptimization") << "For node " << node_i.Id() << " and specified filter radius, maximum number of neighbor nodes (=" << max_number_of_neighbors << " nodes) reached!" << std::endl;
```

Comment:
In such cases I would specify the class or function name as label, to allow for an improved debugging.

Reply:
Will add a more detailed label for warnings.